next/modules/sections/PartnersSection.tsx

```diff
@@ -19,29 +19,29 @@ const PartnersSection = () => {
       {data?.featuredPartners && data.featuredPartners.data.length > 0 && (
         <ul className="mt-6 grid grid-cols-1 gap-5 md:grid-cols-3">
           {data.featuredPartners.data.map((partner, index) => (
-            <PartnerCardRow
-              // eslint-disable-next-line react/no-array-index-key
-              key={index}
-              id={`featured-partner-${index}`}
-              title={partner?.attributes?.title ?? ''}
-              logo={partner?.attributes?.logo?.data?.attributes?.url ?? ''}
-              linkHref={partner?.attributes?.url ?? '#'}
-              featured
-            />
+            <li key={partner?.id ?? index}>
+              <PartnerCardRow
+                id={`featured-partner-${index}`}
+                title={partner?.attributes?.title ?? ''}
+                logo={partner?.attributes?.logo?.data?.attributes?.url ?? ''}
+                linkHref={partner?.attributes?.url ?? '#'}
+                featured
+              />
+            </li>
           ))}
         </ul>
       )}
 
       {data?.notFeaturedPartners && data.notFeaturedPartners.data.length > 0 && (
         <ul className="mt-12 flex flex-col lg:space-y-3">
           {data.notFeaturedPartners.data.map((partner, index) => (
-            <PartnerCardRow
-              // eslint-disable-next-line react/no-array-index-key
-              key={index}
-              id={`non-featured-partner-${index}`}
-              title={partner?.attributes?.title || ''}
-              linkHref={partner?.attributes?.url || ''}
-            />
+            <li key={partner?.id ?? index}>
```

Comment:
Please do not mix it - use either `event.id` if it's unique and defined for all events, or only `index` if it's not.